### PR TITLE
Use Cloud SQL for accounts-db

### PR DESCRIPTION
**Pull request by cloud-pharaoh**
Change summary:
- Replace accounts-db with Cloud SQL database
- Added documentation
- Added link in main README to new deployment option

### CODE REVIEW

File: docs/cloudsql_wi.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```bash
+gcloud sql databases create accounts-db --instance=$INSTANCE_NAME
+```
+5. Install the Cloud SQL proxy by following the instructions [here](https://cloud.google.com/sql/docs/mysql/sql-proxy#install)
+Note: if you are running this from Cloud Shell you can skip this step.)
+6. Connect to the database
+```bash
+gcloud beta sql connect $INSTANCE_NAME --project=$PROJECT_ID --user=accounts-admin --database=accounts-db
+```
+7. Run the SQL commands in [0-accounts-schema.sql](../src/accounts-db/initdb/0-accounts-schema.sql) script on your database to create the tables.
```
Comment (bharathkkb) [inline docs/cloudsql_wi.md:34]: I had actually worked on this before (cant seem to find my branch anymore) and I think we were wondering if we can use an k8s job to make this more easier, although in my branch we were actually using KCC to spin up infra too.

File: extras/cloudsql/userservice.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          periodSeconds: 5
+          timeoutSeconds: 10
+        resources:
+          requests:
+            cpu: 100m
+            memory: 64Mi
+          limits:
+            cpu: 500m
+            memory: 256Mi
+      - name: cloud-sql-proxy
```
Comment (bharathkkb) [inline extras/cloudsql/userservice.yaml:73]: We can squash both of these sidecars into a single deployment for CSQL proxy and have that autoscale according to load.

File: docs/cloudsql_wi.md (hunk 1 above)
Comment (askmeegs) [reply docs/cloudsql_wi.md:34]: Hey bharath, apologies - we did a branch cleanout during the v0.3.0 cleanup/bug fix milestone last month, deleting any branches that were older than a few months.

**Comment by askmeegs** [comment]
Hi Amina, thanks so much for putting together this PR! Unfortunately our Deploy Tests only work on direct branches of this repo, not forks, due to Github actions restrictions on secrets not carrying over into forks. Would it be possible to put out this PR on a new upstream repo branch? 

I will test this PR in the meantime.

**Comment by cloud-pharaoh** [comment]
Closing this in favor of a PR for a branch on upstream -  #383

**Pull request closed without merging**